frontend/canvas-chrome-ext/webpack.config.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Webpack for building the project and creating static
+ * versions of the components we need to display on the webpage and
+ * placing them in the dist folder. The names created here are referenced in
+ * manifest.json
+ *
+ * @type {path.PlatformPath | path}
+ */
+
+const path = require('path');
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const CopyPlugin = require("copy-webpack-plugin");
+
+module.exports = {
+    entry: {
+        popup: './src/popup.jsx',
+        background: './src/background.jsx',
+        studentContent: './src/scripts/student-content.jsx',
+        studentInject: './src/scripts/student-inject-script.jsx',
+        instructorContent: './src/scripts/instructor-content.jsx',
+        instructorInject: './src/scripts/instructor-inject-script.jsx',
```

Comment:
I think it would be less confusing to have the build files and source code files use the same naming convention

Reply:
I tried, dashes are not liked in this file >.<